updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
 
 .Prerequisites
 
+* Have access to a conatiner registry in the disconnected environment to push and pull images. The container registry should be compatible with Docker registry API v2.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[IMPORTANT]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+This is because `oc` command enhancement is not available in previous releases.
+====
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+[id="update-airgapped-env"]
+== Updating an air gapped {product-title} cluster
+
+An air gapped environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap. It may also be the case that there are multiple clusters within the air gapped network. In such a case it makes sense to configure a central registry in the air gapped environment from which every cluster can pull required images for upgrade.
+
+.Procedure
+
+. Findout if there is link:https://access.redhat.com/solutions/4583231[an update path] exists between your current version and the version you want the cluster to move.
```

Comment:
I think Knowledgebase articles are a great stopgap while formal docs are worked out, but I don't like having the formal docs link out to them.  That sort of thing eventually ends up in things like #18944, where we spend a month and many people-hours hashing out how to bring the content into openshift-docs to ease future maintenance.  For the purpose of this PR, can we assume that the user already knows (e.g. from errata) which pullspec they want to mirror and that they don't care about validating the update vs. our upgrade recommendation service (i.e. Cincinnati)?  Because I don't think we have a good official story around figuring out the best next-hop yet (although [this script][1] is in an official-ish repository and automates at least some of what the referenced Knowledgebase article is covering).

[1]: https://github.com/openshift/cincinnati/blob/master/hack/available-updates.sh

Reply:
It is very clear that we have a gap in the user experience workflow around how users can find the right version easily. My plan is to keep the KB article here as it is easier to find this way and then replace it with a official way (with a better user experience). I am planning to create a Jira to track this down.